tools/README.md

```diff
@@ -1,58 +1,20 @@
 # Open-CMSIS-Pack Tools
 
-These command-line tools provide the foundation for Open-CMSIS-Pack-based software development flows and are part of
-the [**CMSIS-Toolbox**](https://github.com/Open-CMSIS-Pack/cmsis-toolbox).
+These command-line tools provide the foundation for Open-CMSIS-Pack-based software development flows and are part of the [**CMSIS-Toolbox**](https://github.com/Open-CMSIS-Pack/cmsis-toolbox).
 
-**Package creation and validation**:
+## SVD file validation
 
-- [**packgen**](./packgen): create a software pack from a `CMake` based software repository.
-- [**packchk**](./packchk): semantic validation of a software pack description and the archive content.
+- [**svdconv**](https://open-cmsis-pack.github.io/svd-spec/main/svd_SVDConv_pg.html) validates CMSIS-SVD files and generates CMSIS-compliant device header files.
 
-**Package management** including discovery of components, devices, boards and examples:
+## Package creation and validation
```

Comment:
I think it is worth mentioning the 
`gen_pack` https://github.com/Open-CMSIS-Pack/gen-pack/?tab=readme-ov-file#bash-library-for-gen-pack-scripts
here as well.